UnitedIncome/components
# Bubble Selector -> TypeScript

### Description
> Please provide a brief description of your changes below.

change Bubble Selector Component to ts.
Added Pure export of icons to allow props to be used in storybook documentation.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

Check to make sure conversion is 1-1

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

### Comments
> Any additional comments that will aid with the review process.
